loadershim.js

```diff
@@ -0,0 +1,3 @@
+global.___loader = {
+    enqueue: jest.fn(),
+  }
```

Comment:
Remove whitespace before closing brace

Reply:
@magnificode these will all disappear once we have prettier set up in #5, too 🙏